actions/docs-verifier/BuildVerifier.IO.Abstractions/BuildVerifier.IO.Abstractions.csproj

```diff
@@ -1,11 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
This should be unneeded. Nullable reference types is enabled in Directory.Build.props:

https://github.com/dotnet/docs-actions/blob/c2790ae335d939e53085b4d9f4a1506de9b95078/actions/docs-verifier/Directory.Build.props#L5

```suggestion
```